conf/distro/nilrt.inc

```diff
@@ -6,23 +6,36 @@ GRUB_BRANCH = "nilrt/19.0"
 
 # Common features enabled on all NILRT flavours
 DISTRO_FEATURES += "\
-        argp \
-        efi \
-        ext2 \
-        largefile \
-        usbgadget \
-        usbhost \
-        wifi \
-        xattr \
-        zeroconf \
-        pci \
-        sysvinit \
-        pam \
-        ptest \
-        selinux \
+	argp \
+	efi \
+	ext2 \
+	largefile \
+	usbgadget \
+	usbhost \
+	wifi \
+	xattr \
+	zeroconf \
+	pci \
+	sysvinit \
+	pam \
+	ptest \
+	selinux \
 	virtualization \
 "
 
+# Clean *most* package work directories
```

Comment:
When recipes are built, bitbake creates a subpackages directory in their workspace, into which it builds the package trees before compressing them into IPKs. Bitbake otherwise doesn't have a good mechanism for determining what packages *actually* resulted from a recipe. Event the `PACKAGES_$PN` variable only tells you what the superset of possible packages are, and once the IPKs are copied into the `deploy/` directory pool, the association with their parent recipe is lost.

So the recipes added to this variable, and the similar variable in local.conf, were added at one time or another because we had some workflow which needed to interrogate that temporary subpackages workspace for the IPKS owned by the recipe.

For this patchset, I've just set these variables to be a superset of what was defined in each of the CG and NXG versions of the pipeline .conf files - for maximum compatibility. In my later patchsets to remove the ARM and NXG components, I will remove entries from the variable which aren't need anymore (which might be all of them).